V2_MIGRATION_GUIDE.md

```diff
@@ -202,6 +202,31 @@ In the following methods the `scope` parameter became non-optional (with a defau
 
 The `multifactorChallenge(mfaToken:types:authenticatorId:)` method lost its `channel` parameter, which is no longer necessary.
 
+### Credentials Manager
+
+`CredentialsManager` now takes a `CredentialsStorage` protocol as it's storage argument rather than an instance of `SimpleKeyChain`.
```

Comment:
```suggestion
`CredentialsManager` now takes a `CredentialsStorage` protocol as it's storage argument rather than an instance of `SimpleKeychain`.
```